Bug: Fix Broken GitHub Workflow

The GitHub workflow for running tests using tox reports errors due to a missing report, see:
```
Run tox -e py
py: install_deps> python -I -m pip install coverage messagebird mypy twilio
.pkg: install_requires> python -I -m pip install 'setuptools>=40.8.0' wheel
.pkg: _optional_hooks> python /opt/hostedtoolcache/Python/3.[7](https://github.com/roaldnefs/django-sms/actions/runs/3728855155/jobs/6324214778#step:5:8).15/x64/lib/python3.7/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
.pkg: get_requires_for_build_sdist> python /opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
.pkg: prepare_metadata_for_build_wheel> python /opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
.pkg: build_sdist> python /opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
py: install_package_deps> python -I -m pip install 'Django>=2.2'
py: install_package> python -I -m pip install --force-reinstall --no-deps /home/runner/work/django-sms/django-sms/.tox/.tmp/package/1/django-sms-0.6.0.tar.gz
py: commands[0]> .tox/py/bin/python -m coverage run setup.py test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
running test
running egg_info
writing django_sms.egg-info/PKG-INFO
writing dependency_links to django_sms.egg-info/dependency_links.txt
writing requirements to django_sms.egg-info/requires.txt
writing top-level names to django_sms.egg-info/top_level.txt
reading manifest file 'django_sms.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'README.Django'
warning: no previously-included files matching '__pycache__' found under directory '*'
warning: no previously-included files matching '*.py[co]' found under directory '*'
adding license file 'LICENSE'
adding license file 'LICENSE.django'
writing manifest file 'django_sms.egg-info/SOURCES.txt'
running build_ext
System check identified no issues (0 silenced).
..........
----------------------------------------------------------------------
Ran 10 tests in 0.151s

OK
/home/runner/work/django-sms/django-sms/.tox/py/lib/python3.7/site-packages/coverage/control.py:[8](https://github.com/roaldnefs/django-sms/actions/runs/3728855155/jobs/6324214778#step:5:9)25: CoverageWarning: No data was collected. (no-data-collected)
  self._warn("No data was collected.", slug="no-data-collected")
py: commands[1]> .tox/py/bin/python -m coverage report
No data to report.
py: exit 1 (0.10 seconds) /home/runner/work/django-sms/django-sms> .tox/py/bin/python -m coverage report pid=2022
.pkg: _exit> python /opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
  py: FAIL code 1 (14.4[9](https://github.com/roaldnefs/django-sms/actions/runs/3728855155/jobs/6324214778#step:5:10)=setup[12.52]+cmd[1.87,0.[10](https://github.com/roaldnefs/django-sms/actions/runs/3728855155/jobs/6324214778#step:5:11)] seconds)
  evaluation failed :( ([14](https://github.com/roaldnefs/django-sms/actions/runs/3728855155/jobs/6324214778#step:5:15).96 seconds)
Error: Process completed with exit code 1.
```

More detailed logging is available on: https://github.com/roaldnefs/django-sms/actions/workflows/tests.yml.

--- a/tox.ini
+++ b/tox.ini
@@ -11,7 +11,7 @@ envlist =
 
 [testenv]
 commands =
-    {envpython} -m coverage run setup.py test
+    {envpython} -m coverage run --source=sms setup.py test
     {envpython} -m coverage report
     {envpython} -m coverage html
     {envpython} -m mypy --config-file=tox.ini sms tests
@@ -35,8 +35,7 @@ commands =
 deps=
     flake8
 
-[coverage:run]
-include = sms*
+[coverage]
 
 [mypy]
 

